torchdp/test/privacy_engine_test.py

```diff
@@ -163,8 +161,8 @@ def test_throws_double_attach(self):
         model, optimizer = self.setUp_init_model(private=True)
         self.setUp_model_step(model, optimizer)
         with self.assertRaises(ValueError):
-            model, optimizer = self.setUp_init_model(private=True, model=model)
-            self.setUp_model_step(model, optimizer)
+            model, optimizer2 = self.setUp_init_model(private=True, model=model)
```

Comment:
was there a bug in the test before this change?

Reply:
Good catch! This test had to be changed in a previous design of virtual batches. With the current design, the original test passes.